concepts/05 UI Components/Form/01 Getting Started with Form/70 Validate the Form Data.md

```diff
@@ -0,0 +1,172 @@
+DevExtreme includes a validation engine that checks edited values before they are saved. This engine supports different [validation rule types](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). 
```

Comment:
The first sentence says "before they are saved". A sentence in the next paragraph says "an editor is validated when its value changes". I'm not sure which is right... Or maybe those two sentences talk about different things?

Reply:
The validation cycle is the following:
1. A user types/selects something.
2. The value changes.
3. The user focuses the editor out or presses the 'Enter' button.
4. The validation rule(s) triggers.
5. If the checks pass successfully, the value is saved, Otherwise, the red icon and an error messages (if any) appears.

Reply:
So then the following phrase in the other paragraph is wrong: "an editor is validated when its value changes". Right?